integration/microservices/e2e/sum-kafka.spec.ts

```diff
@@ -11,27 +11,11 @@ import { Test } from '@nestjs/testing';
 import { expect } from 'chai';
 import * as request from 'supertest';
 import { KafkaController } from '../src/kafka/kafka.controller';
-import { APP_FILTER } from '@nestjs/core';
-import { Observable, throwError } from 'rxjs';
 import { KafkaMessagesController } from '../src/kafka/kafka.messages.controller';
 import { UserDto } from '../src/kafka/dtos/user.dto';
 import { UserEntity } from '../src/kafka/entities/user.entity';
 import { BusinessDto } from '../src/kafka/dtos/business.dto';
-import { BusinessEntity } from '../src/kafka/entities/business.entity';
-
-@Catch()
-class KafkaExceptionFilter implements ExceptionFilter {
-  catch(exception: HttpException, host: ArgumentsHost): any {
-    console.log(exception);
-  }
-}
-@Catch()
-class RpcErrorFilter implements RpcExceptionFilter {
-  catch(exception: RpcException): Observable<any> {
-    console.log(exception);
-    return throwError(exception);
-  }
-}
+import * as async from 'async';
```

Comment:
How about using Promise.all for concurrency instead of async?

Reply:
Yeah, we can refactor that, i used this async parallel before, and promise.all before, they both do the same i think, but its more prettier use the Promise.all :D

Reply:
Would much rather use Promises or Observables.  Also I don't think async is a nestjs npm dependency.

Reply:
@mkaufmaner its done